.github/workflows/build.yml

```diff
@@ -87,8 +88,15 @@ jobs:
           TEST_COVERAGE: 1
         run: |
           make test
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          file: ./out/tests/coverage-unit.txt
+          flags: unit,os_windows
+          fail_ci_if_error: true
+          verbose: true
       - name: Prepare Codecov
-        uses: crazy-max/ghaction-chocolatey@v1
+        uses: crazy-max/ghaction-chocolatey@v2
```

Comment:
bump to v2 (mirroring what's in buildpacks/pack)

Reply:
TBH I found the split here between linux and windows a bit confusing or I guess I don't really understand it. I'm wondering if only windows installs codecov, both run tests (maybe the linux one is ignored bc not installed?), and only the linux section uploads the coverage-unit.txt to hosted codecov..?

Or is perhaps something implicit happening here that I'm missing?

Reply:
I'm a bit fuzzy on this myself TBH... I do see output from Codecov in the linux workflows, but I'm not sure why it doesn't need to be installed explicitly.